.eslintrc.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
       },
       plugins: ['node'],
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
-        // add your custom rules and overrides for node files here
+        'node/no-unpublished-require': 'off',
```

Comment:
Why?

Reply:
This is a bit heavy handed, but this triggered the no-unpublished-require rule because ember-cli-version-checker is a devDependency, but is published. I should likely either target the override to just this file, or move the dependency to dependencies in package.json.

Reply:
Fixed.